src/components/ChainSelectionPopover/ChainSelectionPopover.styles.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from "styled-components/macro";
+
+import breakPoints from "../../style/breakpoints";
+import { BorderlessButtonStyle } from "../../style/mixins";
+
+/**
+ * @remarks props are used to manipulate the positioning of components
+ */
+type ContainerProps = {
+  open: boolean;
+  shiftLeft: boolean;
+};
+
+export const Container = styled.div<ContainerProps>`
+  position: absolute;
+  display: grid;
+  grid-template-rows: 5rem auto 5.125rem;
```

Comment:
Only use grid when there's an actual grid. Display flex works just fine in this case.